tiatoolbox/utils/misc.py

```diff
@@ -453,6 +454,36 @@ def conv_out_size(in_size, kernel_size=1, padding=0, stride=1):
     return (np.floor((in_size - kernel_size + (2 * padding)) / stride) + 1).astype(int)
 
 
+def parse_cv2_interpolaton(interpolation: Union[str, int]) -> int:
+    """Convert a string to a OpenCV (cv2) interpolation enum.
+
+    Valid integer values for cv2 INTER_ enums are passed through.
+
+    Args:
+        interpolation (Union[str, int]): Interpolation mode string.
+            Possible values are: neares, linear, cubic, lanczos, area.
+
+    Raises:
+        ValueError: Invalid interpolation mode.
+
+    Returns:
+        int: OpenCV (cv2) interpolation enum.
```

Comment:
Can we add links to opencv documentation as with :class:numpy ? I guess not as opencv is basically c/c++

Reply:
Can probably link to the cv2 python docs. I'll have a look.